toffy/normalize.py

```diff
@@ -19,6 +19,9 @@
 from mibi_bin_tools.bin_files import extract_bin_files, get_median_pulse_height
 from mibi_bin_tools.panel_utils import make_panel
 
+from toffy.json_utils import read_json_file
```

Comment:
Same thing here, and in the other files in this PR where you import both; cleaner to put on a single line